src/Resources/translations/SonataAdminBundle.de.xliff

```diff
@@ -9,6 +9,14 @@
                 <source>sonata_media</source>
                 <target>Medienbibliothek</target>
             </trans-unit>
+            <trans-unit id="form.label_icon">
+                <source>form_group.gallery</source>
+                <target>form_group.gallery</target>
```

Comment:
User will see the translation changing from `Gallery` to `form_group.gallery`

Reply:
Yep, how do you prefer to handle this, We have 2 options:

1. We add translations to all languages with the placeholder
2. We dont add the translation on all languages.

Doing 1 is good because is what is done on AdminBundle (and I believe this was already discussed)

Doing 2 is good because on other languages if you have 'en' as default. But, you will end with  a lot of files of translations without same keys, a mess.

I was following 1. and my intention was to add all the keys to all languages on 3.x.

But not strong opinion on any, just would be nice to know how to proced with translations.

Reply:
There is a general discussion https://github.com/sonata-project/dev-kit/issues/751

> Doing 1 is good because is what is done on AdminBundle (and I believe this was already discussed)

I solved the issues by replacing every placeholder by the translation.

> Doing 2 is good because on other languages if you have 'en' as default. But, you will end with a lot of files of translations without same keys, a mess.

I agree, when I tried to add the translations on SonataAdmin, it was a mess to understand which was missing and which was not. Looking at symfony process https://github.com/symfony/symfony/issues/38710
- There is a website with the status of translations https://symfony-translations.nyholm.tech/
- Issues are created for this https://github.com/symfony/symfony/issues/42315
- They are using number as `id` (so it's easy to check the order)

In our case I would say
- Seing on the dev-kit site the status of the translations
- Having a CI-check about the order of the translations (every files should use the same order than the `en` file).
would be the requirements to avoid this "mess".

Since I couldn't decide between 1 and 2, I added translations in every local when I need some ^^'

Reply:
"Galerie"

Reply:
Fixed the "de" translations, but the conversation is still going.

Reply:
If we are going to use Weblate, for now I will remove those placeholders, because using weblate it is better to not add fake translations, to keep a better track of the current status.